hs_core/views/__init__.py

```diff
@@ -60,7 +61,8 @@
 
 from hs_collection_resource.models import CollectionDeletedResource
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('hydroshare')
+# logger = logging.getLogger(__name__)
```

Comment:
remove the commented out logger statement. I think the original logging.getLogger(__name__) should work better rather than the hardcoded "hydroshare" logger name string.

Reply:
Agreed. Will fix.